rules/code-of-conduct.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const find = require('unist-util-find');
+const rule = require('unified-lint-rule');
+const findAuthorName = require('../lib/find-author-name');
+
+const authorName = 'sindresorhus';
+const authorEmail = 'sindresorhus@gmail.com';
+
+module.exports = rule('remark-lint:awesome/code-of-conduct', (ast, file) => {
+	if (ast.children.length === 0) {
+		file.message('code-of-conduct.md file must not be empty');
```

Comment:
No need to mention the file name. That will be in the report anyway.
```suggestion
		file.message('The file must not be empty');
```

Reply:
I followed the same behavior with `awesome/contributing` rule. It reports like,

- Missing file contributing.md
- contributing.md file must not be empty

Personally, I think it is better. Because from the rule name, you can not know the required file name exactly.

Reply:
Good point